examples/Assertions/Marking/test_plan.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python
```

Comment:
This is a filing example, suggest to turn it into a passing one with passing assertions, or log assertions. That way the example at least test if it is running without a problem. However this is a nice deterministic testsuite, the line numbers can be validated in a more unit/functional level of test.

Reply:
Is your suggestion to add unit test to see whether we exactly point to where we want and turn the example into something that passes everywhere? I do like the visuality of the all failing because it immediately tells me where I wanted to look.

That said I am OK with that, just asking for clarification.